src/constants.ts

```diff
@@ -60,5 +72,6 @@ export const MIGRATIONS_DIR = path.join(
 export const CONTENTFUL_ALIAS = INPUT_CONTENTFUL_ALIAS || DEFAULT_CONTENTFUL_ALIAS;
 export const DELAY = Number(INPUT_DELAY || DEFAULT_DELAY);
 export const MAX_NUMBER_OF_TRIES = Number(INPUT_MAX_NUMBER_OF_TRIES || DEFAULT_MAX_NUMBER_OF_TRIES);
-export const CREATE_CDA_TOKEN = booleanOr(INPUT_CREATE_CDA_TOKEN, DEFAULT_CREATE_CDA_TOKEN);
 export const DEFAULT_BRANCH_NAME = INPUT_DEFAULT_BRANCH_NAME || null;
+
+export const ACTIONS = INPUT_ACTIONS?.trim()?.split(/,\s*/) ?? DEFAULT_ACTIONS;
```

Comment:
curious: why does it have to be split with `/.\s*/`? won't this also match the word after the comma?

should we have a simple function for this with a supporting unit test to prove it does what it should?

Reply:
`\s` is the whitespace regex, so this is to eat any whitespace after a comma. A lazy man's trim done during a split. I _should_ protect from trailing spaces before the comma, but it's an internal tool, can't do too much..